signin/signin.js

```diff
@@ -0,0 +1,85 @@
+const emailInput = document.querySelector(".email-input");
+const pwInput = document.querySelector(".password-input");
+const pwWrapper = document.querySelector(".password-wrapper");
+const signinBtn = document.querySelector(".signin-btn");
+const eyeBtn = document.querySelector(".eye-off-btn");
+const emailEmptyMsg = document.createElement("span");
+const emailInvalidMsg = document.createElement("span");
+const pwEmptyMsg = document.createElement("span");
+
+let regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
```

Comment:
```suggestion
let regex = new RegExp("[a-z0-9]+@[a-z]+\\.[a-z]{2,3}");
```
온점이 없어도 현재는 통과하는데요, 여기서 이스케이프를 하지 않으면 정규표현식에 따라 온점은 any character로 인식됩니다. vscode prettier 가 자동으로 포매팅 해주면서 없어지는 이슈라면 저렇게 이스케이프 두 번 해주면 된다고 합니다~